metricbeat/module/logstash/docker-compose.yml

```diff
@@ -2,22 +2,22 @@ version: '2.3'
 
 services:
   logstash:
-    image: docker.elastic.co/integrations-ci/beats-logstash:${LOGSTASH_VERSION:-7.12.0}-1
+    image: docker.elastic.co/integrations-ci/beats-logstash:${LOGSTASH_VERSION:-8.1.0}-1
```

Comment:
As this is the module collecting stats, I wonder if the tests should really be against a snapshot / unreleased version or rather a released version?

Reply:
Can anyone confirm if this image definitely even exists? https://container-library.elastic.co/r/integrations-ci/beats-logstash gives me a 404 even when signed in via elastic okta.

Reply:
I think this is built from the local Dockerfile: https://github.com/elastic/beats/blob/master/metricbeat/module/logstash/_meta/Dockerfile See the path to _meta here: https://github.com/elastic/beats/blob/master/metricbeat/module/logstash/docker-compose.yml#L7

Reply:
Ah, I see. I guess the `image` line just gives it a local name then. Never used both build & image in the same file I don't think.